### PR TITLE
New package: tmfs-r5

### PR DESCRIPTION
**Pull request by guinanseyebrows**

### CODE REVIEW

**Comment by jgouly** [comment]
LGTM!

File: srcpkgs/tmfs/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'tmfs'
+pkgname=tmfs
+version=r5
+revision=1
+build_style=cmake
+hostmakedepends="cmake pkg-config"
+makedepends="fuse-devel boost-devel"
+depends="fuse boost"
```
Comment (xtraeme) [inline srcpkgs/tmfs/template]: This `depends` line is completely wrong.

File: srcpkgs/tmfs/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+short_desc="Apple Time Machine read-only mounting utility"
+maintainer="sth <sth@nvsbl.org>"
+license="MIT"
+homepage="https://github.com/abique/tmfs"
+distfiles="https://github.com/abique/tmfs/archive/${version}.tar.gz"
+checksum="897ad09e21ddd8e912dc1baba4466d8a65a09b3abdeb31e4640c2d770915f2f4"
+only_for_archs="x86_64 i686"
```
Comment (xtraeme) [inline srcpkgs/tmfs/template]: This `only_for_archs` line is completely wrong.

**Comment by guinanseyebrows** [comment]
waiting on boost update before this can be built on armv6/7hf 

curently set to only_for_archs="x86_64 i686 armv6hf armv7hf x86_64-musl" but this should be removed when boost is fixed.

**Comment by guinanseyebrows** [comment]
now builds successfully on all archs

**Pull request merged**